README.md

```diff
@@ -56,10 +55,11 @@ We also have packages that are still in their planning stages and as such might
 * [`dojo/app`](https://github.com/dojo/app) - An application framework for Dojo 2
 * [`dojo/crypto`](https://github.com/dojo/crypto) - A set of libraries to deal with cryptography
 * [`dojo/dataviz`](https://github.com/dojo/dataviz) - A data visualization library for Dojo 2
-* [`dojo/dom`](https://github.com/dojo/dom) - A set of APIs for manipulating the DOM
 
 We have some deprecated packages:
 
+* [`dojo/compose`](https://github.com/dojo/compose) - A mixin/trait based composition library
+* [`dojo/dom`](https://github.com/dojo/dom) - A set of APIs for manipulating the DOM
```

Comment:
Feels like we should have a sentence explaining why it was deprecated like we did for parser